awx/main/models/events.py

```diff
@@ -204,6 +204,12 @@ class Meta:
         default=0,
         editable=False,
     )
+    created = models.DateTimeField(
```

Comment:
This is going to potentially be problematic (really slow) to add for people with loooooooooooooots of job event entries.

That said, we *do* need this for the analytics queries we want to do.

@matburt what say you.

Reply:
Yeah this alter table is going to be big, but I don't necessarily see a way around it. The index itself is going to be ~1GB per 100 million events.

Reply:
It might be worth mentioning this in the release notes, that the upgrade will take a little while.

Reply:
@rooftopcellist tried one of these on an `m3.large` w/ 120M rows and it took about 5 minutes.  I'm happy with this.